libexec/bats-core/bats

```diff
@@ -26,14 +26,17 @@ Usage: $cmd [-cr] [-f <regex>] [-j <jobs>] [-p | -t] <test>...
   <test> is the path to a Bats test file, or the path to a directory
   containing Bats test files (ending with ".bats").
 
-  -c, --count      Count the number of test cases without running any tests
-  -f, --filter     Filter test cases by names matching the regular expression
-  -h, --help       Display this help message
-  -j, --jobs       Number of parallel jobs to run (requires GNU parallel)
-  -p, --pretty     Show results in pretty format (default for terminals)
-  -r, --recursive  Include tests in subdirectories
-  -t, --tap        Show results in TAP format
-  -v, --version    Display the version number
+  -c, --count               Count the number of test cases without running any tests
+  -e, --extra-flags         Extra flags to pass to the bats executor (Options: -T)
+  -f, --filter              Filter test cases by names matching the regular expression
+  -F, --formatter           Switch between formatters (Default: bats-format-tap, Options: bats-format-junit, bats-format-pretty, bats-format-tap)
+  -h, --help                Display this help message
+  -j, --jobs                Number of parallel jobs to run (requires GNU parallel)
+  -o, --output              A directory to output reports to
+  -p, --pretty              [DEPRECATED] Show results in pretty format (default for terminals), use "--formatter bats-format-pretty" instead
```

Comment:
This is a pretty major flag to deprecate, and introduces something that will force a breaking change to remove. I err on the side of the sysadmin, and would prefer to not deprecate flags that will break scripts in the future when removed.